tests/test_pyglmnet.py

```diff
@@ -110,12 +94,33 @@ def test_multinomial():
     # uniform prediction
     ynull = np.ones(yhat.shape) / yhat.shape[1]
     # pseudo_R2 should be greater than 0
-    assert_true(glm_mn.score(y, yhat, ynull, method='pseudo_R2') > 0.)
-    glm_mn.score(y, yhat)
+    assert_true(glm_mn.general_score(y, yhat, ynull, method='pseudo_R2') > 0.)
+    glm_mn.general_score(y, yhat)
     assert_equal(len(glm_mn.simulate(glm_mn.fit_[0]['beta0'],
                                   glm_mn.fit_[0]['beta'],
                                   X)),
                  X.shape[0])
     # these should raise an exception
-    assert_raises(ValueError, glm_mn.score, y, y, y, 'pseudo_R2')
-    assert_raises(ValueError, glm_mn.score, y, y, None, 'deviance')
+    assert_raises(ValueError, glm_mn.general_score, y, y, y, 'pseudo_R2')
+    assert_raises(ValueError, glm_mn.general_score, y, y, None, 'deviance')
+
+
+def test_sklearn_compatibility():
+    X, y = make_regression(n_samples=1000, n_features=10)
+
+    # Test whether cross validation works
+    glm_normal = GLM(distr='normal', alpha=0.01,
+                     reg_lambda=[0.0, 0.1, 0.2])
+    glm_normal.fit(X, y)
+    cv = KFold(X.shape[0], 5)
+    # check that it returns 5 scores
+    assert_equal(len(cross_val_score(glm_normal, X, y, cv=cv,
+                                     scoring=simple_cv_scorer)), 5)
+
+    # Test sklearn pipelines
+    scaler = StandardScaler()
+    glm = GLM(distr='normal', alpha=0.01,
+              reg_lambda=[0.0, 0.1, 0.2])
+    clf = Pipeline([("scaler", scaler), ("glm", glm)])
+    assert_equal(len(cross_val_score(clf, X, y, cv=cv,
+                                     scoring=simple_cv_scorer)), 5)
```

Comment:
the `score` function of `GLM` returns the score for each regularization parameter, so we need to combine those scores because `cross_val_score` expects only one number. We probably need to decide whether `score` should return some sort of average and be totally compatible with `sklearn`.